runtime/interpreter/decode.go

```diff
@@ -81,6 +82,17 @@ func decodeString(dec *cbor.StreamDecoder, memoryGauge common.MemoryGauge) (stri
 	return dec.DecodeString()
 }
 
+func decodeBigInt(dec *cbor.StreamDecoder, memoryGauge common.MemoryGauge) (*big.Int, error) {
```

Comment:
Maybe make this a receiver-method of `StorableDecoder`, so the `*cbor.StreamDecoder` and `memoryGauge` can be obtained from the receiver

Reply:
Good idea! 

I replicated `decodeString`, which is called from different decoders (storable, type, location), but you are right `decodeBigInt` is only ever called from `StorableDecoder` 👍 

Refactored in d47975221eb129b00395f9f03a5b477b23567472